tls/Makefile

```diff
@@ -24,7 +24,13 @@ EXTRA_CFLAGS += -Wframe-larger-than=4096
 
 obj-m = tempesta_tls.o
 
-ttls-srcs = $(wildcard $(obj)/*.c)
+ttls-srcs = $(wildcard $(src)/*.c)
 ttls-objs = $(patsubst %.c, %.o, $(ttls-srcs))
 
+# exclude some original files (see tempesta_fw/ for re-implementation)
+ttls-objs := $(filter-out	\
+	$(src)/ssl_tls.o	\
+	$(src)/ssl_srv.o	\
+	, $(ttls-objs))
```

Comment:
No, that's just wrong! Firstly, we should not keep the same source code in `tls` and `tempesta_fw` at the same time. Secondly, all the code in `tempesta_fw` must obey to our coding style. Next, you should have been accurately split the logic between `tls` and `tempesta_fw`, not just copy & paste source code.